### DISCUSSION TITLE
Is it pssible to hide one partcular lang from the list?

### DISCUSSION THREAD

**Discussion by Linux-Server** [anuraghazra/github-readme-stats#3616] [Q&A] [answered]
This is my most used lang list. Is it possible to hide makerfile from the most used lang list from below link
`https://github-readme-stats.vercel.app/api/top-langs?username=linux-server&show_icons=true&langs_count=10&locale=en&layout=donut-vertical&theme=dark#gh-dark-mode-only`

**Comment by qwerty541** [comment, marked as answer]
Hey, Sachin! You can hide the language from your card using `&hide=makefile` query option. Here is an example: 

```html
<img src="https://github-readme-stats.vercel.app/api/top-langs?username=linux-server&show_icons=true&langs_count=10&locale=en&layout=donut-vertical&theme=dark&hide=makefile#gh-dark-mode-only" />
```

<img src="https://github-readme-stats.vercel.app/api/top-langs?username=linux-server&show_icons=true&langs_count=10&locale=en&layout=donut-vertical&theme=dark&hide=makefile#gh-dark-mode-only" />

**Reply by Linux-Server** [reply]
Thank you
